cabby/model/text/dual_encoder/dataset_rvs.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple, Sequence, Optional, Dict, Text, Any
+
+from absl import logging
+import numpy as np
+import os
+import pandas as pd
+import sys
+
+import torch
+import transformers
+from transformers import DistilBertTokenizerFast
+from transformers.tokenization_utils_base import BatchEncoding
+
+from shapely.geometry.point import Point
+
+from cabby.geo import util as gutil
+from cabby.model.text import util 
+
+from cabby.geo import regions
+from cabby.model.text.dual_encoder import dataset_item
+
+
+tokenizer = DistilBertTokenizerFast.from_pretrained('distilbert-base-uncased')
+
+CELLID_DIM = 64
+
+
+
+class TextGeoSplit(torch.utils.data.Dataset):
```

Comment:
Please add a reasonably comprehensive docstring to this class, indicating what it is for and what it comprises.